__tests__/app.test.js

```diff
@@ -350,3 +350,32 @@ describe("DELETE /api/comments/:comment_id", () => {
       });
   });
 });
+describe("GET /api/users", () => {
+  test("should be available on /api/users", () => {
+    return request(app).get("/api/users").expect(200);
+  });
+  test("should response with an articles array of users objects, each of which should have the expected properties", () => {
+    return request(app)
+      .get("/api/users")
+      .expect(200)
+      .then(({ body }) => {
+        const { users } = body;
+        expect(users).toHaveLength(4);
+        users.forEach((user) => {
+          expect(user).toMatchObject({
+            username: expect.any(String),
+            name: expect.any(String),
+            avatar_url: expect.any(String),
+          });
+        });
+      });
+  });
+  test("status:404, responds with an error message when route does not exist", () => {
```

Comment:
This test isn't specific to this endpoint, it's just a test for checking an incorrect endpoint. You can either remove it, or put it in its own describe block 👍